Backend/src/database/services/user.service.ts

```diff
@@ -18,6 +18,29 @@ export default class UserService {
     return await userModel.find().exec();
   }
 
+  static async searchUsers(keyword: string | null | undefined, sortBy: string | null | undefined) {
+    //TODO implement actual fuzzy searching and other sorting methods
+    let sort = 1;
+    if (sortBy == "namedesc") {
+      sort = -1;
+    }
+    let keywordFilter: FilterQuery<IUser> = {};
+    if (keyword && keyword.length > 0) {
+      keywordFilter = { $or: [{ firstName: keyword }, { lastName: keyword }] };
+    }
+    return await userModel
+      .find(keywordFilter, {
```

Comment:
Are you only returning the id of the found user here?

Reply:
Only non-private info like Name avatar and note. Excluding unnecessary/private info.